data/mods/obsolete-mods.json

```diff
@@ -1 +1 @@
-[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines" ]
+[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines", "No_Diamond_Weapons" ]
```

Comment:
Rather than obsoleting the mod, you should make it set an option to reroll CVD mapgen results.